brave/brave-ios
# Fix #5140: Add 1.37 translations.

## Summary of Changes


This pull request fixes #5140 

## Submitter Checklist:


## Test Plan:

- Go to a reader mode, verify that the `Reader Mode` label below url bar is translated
- Go to appearance settings, verify that 'night mode' related text is translated

## Screenshots:



## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `bug` / `enhancement`
